packages/waku/src/lib/builder/build.ts

```diff
@@ -696,6 +718,7 @@ export async function build(options: {
     config,
     distEntries,
     buildConfig,
+    options.partial || false,
```

Comment:
For consistent coding style,
```suggestion
    !!options.partial,
```